api/src/main/java/uk/gov/dhsc/htbhf/eligibility/controller/v2/EligibilityControllerV2.java

```diff
@@ -0,0 +1,46 @@
+package uk.gov.dhsc.htbhf.eligibility.controller.v2;
+
+import lombok.AllArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseBody;
+import uk.gov.dhsc.htbhf.dwp.model.v2.IdentityAndEligibilityResponse;
+import uk.gov.dhsc.htbhf.dwp.model.v2.PersonDTOV2;
+import uk.gov.dhsc.htbhf.eligibility.model.v1.PersonDTO;
+
+import javax.validation.Valid;
+
+import static org.springframework.http.MediaType.APPLICATION_JSON_VALUE;
+
+/**
+ * Responsible for obtaining a decision on identity matching and eligibility from downstream services, sending the response
+ * back to the claimant service and returning the result.
+ */
+@RequestMapping("/v2/eligibility")
+@Controller
+@Slf4j
+@AllArgsConstructor
+public class EligibilityControllerV2 {
+
+    /**
+     * Invokes downstream services to obtain a decision on the identity and eligibility for the given person.
+     *
+     * @param person the {@link PersonDTO}
+     * @return the {@link IdentityAndEligibilityResponse}
+     */
+    @PostMapping(consumes = APPLICATION_JSON_VALUE, produces = APPLICATION_JSON_VALUE)
+    @ResponseBody
+    public ResponseEntity<IdentityAndEligibilityResponse> getIdentityAndEligibilityDecision(@RequestBody @Valid PersonDTOV2 person) {
+        log.debug("Received eligibility request");
+
+        IdentityAndEligibilityResponse response = IdentityAndEligibilityResponse.builder().build();
+
+        return new ResponseEntity<>(response, HttpStatus.OK);
```

Comment:
Any reason we're manually creating a response entity rather than just returning the IdentityAndEligibilityResponse object?

Reply:
When the controller is updated to call the service, we'll want to change the status code dependant upon the response being sent, so it was in preparation for that, but I can easily remove it if you want.

Reply:
Actually, talking to Matt, we're not going to do this now anyway so I'll remove.